lib/db/files_db.dart

```diff
@@ -807,6 +787,46 @@ class FilesDB {
     return result;
   }
 
+  Future<Set<String>> getLocalFileIDsForCollection(int collectionID) async {
+    final db = await instance.database;
+    final rows = await db.query(
+      filesTable,
+      columns: [columnLocalID],
+      where: '$columnLocalID IS NOT NULL AND $columnCollectionID = ?',
+      whereArgs: [collectionID],
+    );
+    final result = <String>{};
+    for (final row in rows) {
+      if (kDebugMode && result.contains(row[columnLocalID])) {
```

Comment:
Will this ever happen?

Reply:
Ideally no, this check I had added to catch any unexpected db state during dev/debug mode. Will either remove or add a comment saying that we can remove this once rewrite is out and stable.